report-api/report-templates/invoice.html

```diff
@@ -499,8 +499,8 @@
               <p>
                 1. Sign in to your financial institution's online banking website or app<br>
                 2. Go to your financial institution's bill payment page<br>
-                3. Enter <span class="font-bold">"BC Registries and Online Services"</span> as payee<br>
-                4. Enter the following account number: <span class="font-bold">{{accountNumber}}</span><br>
+                3. Enter <span class="font-bold">"BC Registries"</span> as payee<br>
+                4. Enter this payment identifier as your account number: <span class="font-bold">{{accountNumber}}</span><br>
```

Comment:
need to check on account number and how this report is used. Need to update to cfsAccountId if it isn't already